xclim/indicators/land/_streamflow.py

```diff
@@ -16,10 +16,13 @@
 
 
 class Streamflow(ResamplingIndicator):
+    """Streamflow class."""
+
     context = "hydro"
     src_freq = "D"
     keywords = "streamflow hydrology"
 
+    # TODO: TJS: The signature of this method seems wrong. Should it be `def cfcheck(cls, q):` or something else? Is it a static method?
```

Comment:
@aulemahal Any advice on this particular method ? Any way we can make this consistent ?

Reply:
Indeed. all other implementations of `cfcheck` are normal methods. I would agree to do the same here.

However, I'll note that if we solve Eric's comment here https://github.com/Ouranosinc/xclim/pull/1832#discussion_r1725793960, then this function could be removed as the base  `cfcheck` will work.

Reply:
@coxipi Would it make more sense to address this in your PR then? I'll leave the `TODO` where it is.

Reply:
Not my PR, but I will add the relevant information in the PR so that the cfcheck function can be removed once this is fixed.